scipy/linalg/_matfuncs_expm.pyx.in

```diff
@@ -217,9 +217,9 @@ def pick_pade_structure(cnp.ndarray[ndim=3, dtype=numpy_lapack_t] Am):
         normA *= 2.**(-s)
 
     # 1-norm of A ** 27
-    for i in range(8):
-        np.matmul(absA, work_arr, out=work_arr)
-    temp = np.max(work_arr)
+    for i in range(2):
+        np.matmul(absA_pow4, work_arr, out=work_arr)
```

Comment:
This change was not valid, because if `s != 0` the value of `absA` is scaled with `absA *= two_pow_s`. I will correct for this by scaling `absA_pow4` as well. It would be nice of we had a unit test that triggers this.